k4GeneratorsConfig/src/xsection.cxx

```diff
@@ -84,25 +84,26 @@ bool k4GeneratorsConfig::xsection::processFile(){
   }
   unsigned int lastEvent = m_reader->getEntries(podio::Category::Event) - 1;
   auto event = podio::Frame(m_reader->readEntry(podio::Category::Event,lastEvent));
-  
+
   // decode sqrts
-  m_sqrts = event.getParameter<double>("SQRTS");
+  const auto& sqrts = event.getParameter<double>("SQRTS");
+  m_sqrts = sqrts.value_or(0.);
 
```

Comment:
```suggestion
m_sqrts = event.getParameter<double>("SQRTS").value_or(0.);

```
This can be done in one line here, without introducing another temporary.